File: app/src/main/java/br/com/nicetuber/util/UIListeners.java
package br.com.nicetuber.util;

import br.com.nicetuber.model.ChannelSearched;

/**
 * Created by thiagozg on 15/10/2017.
 */

public interface UIListeners {

    interface OnClickListener {
        void onClick(ChannelSearched channelSearched);
    }

}

File: app/src/main/java/br/com/nicetuber/util/UIListeners.kt
package br.com.nicetuber.util

import br.com.nicetuber.model.ChannelSearched

/**
 * Created by thiagozg on 15/10/2017.
 */

interface UIListeners {

    interface OnClickListener {
        fun onClick(channelSearched: ChannelSearched)
    }

}
